pkg/commonvpc/common_test_functionality.go

```diff
@@ -47,11 +37,17 @@ const (
 const errString = "err: %s"
 const carriageReturn = "\r"
```

Comment:
merge with the `const` section above (and there are more consts afterwards that can be merged too)

Reply:
done